app/assets/javascripts/surveys.js

```diff
@@ -22,6 +22,32 @@
       frequency: 50,
     },
     smallSurveys: [
+      {
+        url: 'https://www.surveymonkey.com/s/2MRDLTW',
+        identifier: 'user_satisfaction_survey',
+        template: TEMPLATE,
+        frequency: 10,
+        activeWhen: function() {
+          function breadcrumbMatches() {
+            var text = $('#global-breadcrumb').text() || "";
```

Comment:
This will return `false` (and not error) when the breadcrumb is not present on the page.